### PR TITLE
FAQ.rst: Add item about CoC

### PR DESCRIPTION
**Pull request by PrajwalM2212**
This adds Code of Conduct item to FAQ.

Closes https://github.com/coala/documentation/issues/554

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: Help/FAQ.rst [hunk 1]
```diff
@@ -138,3 +138,83 @@ our master branch. Right now we use 7 of those checks:
 
 - **continuous-integration/appveyor/pr** This one does the same as the one
   above, but for Windows, both 32 and 64bits versions.
+
+What is coala Community Code of Conduct?
+----------------------------------------
+
+Like the technical community as a whole, the coala team and community is
```
Comment (jayvdb) [inline Help/FAQ.rst]: Please dont include the entire CoC.

Only an item about the CoC, so people know we have one, and how to find it.

**Comment by PrajwalM2212** [comment]
@jayvdb  I have made the changes. I have trimmed the CoC to include only the must have details. Do tell me if I need to shorten it even more.

File: Help/FAQ.rst [hunk 2]
```diff
@@ -138,3 +138,31 @@ our master branch. Right now we use 7 of those checks:
 
 - **continuous-integration/appveyor/pr** This one does the same as the one
   above, but for Windows, both 32 and 64bits versions.
+
+What is coala Community Code of Conduct?
+----------------------------------------
+
+The coala team and community is made up of a mixture of
+professionals and volunteers from all over the world.Diversity is
```
Comment (jayvdb) [inline Help/FAQ.rst]: add space after `.`

File: Help/FAQ.rst [hunk 3]
```diff
@@ -138,3 +138,31 @@ our master branch. Right now we use 7 of those checks:
...
+
+- **Be considerate.**
+
+- **Be respectful.**
+
+- **Be careful in the words that you choose.**
+
+- **When we disagree, try to understand why.**
+
+For more information refer `here <http://coala.io/coc>`__.
```
Comment (jayvdb) [inline Help/FAQ.rst]: avoid "here" as a link text.

**Comment by PrajwalM2212** [comment]
@jayvdb I have made the changes.

**Comment by jayvdb** [comment]
ack 08e73b5

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**